internal/cli/deployments/setup.go

```diff
@@ -242,7 +241,8 @@ func (opts *SetupOpts) configureMongod(ctx context.Context) error {
 		return err
 	}
 
-	const healthyDeploymentTimeout = 2 * time.Minute
+	// This can be a high number because the container will become unhealthy before the 10 minutes is reached
+	const healthyDeploymentTimeout = 10 * time.Minute
```

Comment:
this seems very high, is it for both podman and docker?

Reply:
Yes, but we will never reach this timeout because the timeout in our container is set to 2 minutes.
The only case where we will reach this timeout is when podman/docker doesn't run the healthcheck, which should never happen.

Reply:
how about setting this to 3 minutes?

Reply:
sure, will update